src/components/text-input.components.js

```diff
@@ -0,0 +1,26 @@
+import $ from 'jquery';
+import Input from './input-components';
+export default class TextInput extends Input {
+    constructor(stringQuery, required = true, disabled = false) {
+        super(disabled, required);
+        this.stringQuery = stringQuery;
+
+        this._textInput = $(`${stringQuery} input[type="text"]`);
+
+        this._textValid = false;
+    }
+
+    setInput(value) {
+        this._textInput = this._textInput.val(value);
+    }
+
+    isTextValid() {
+        if (this._textInput.val() === '') {
+            this._textValid = false;
+            return this._textValid;
+        }
+
+        this._textValid = true;
+        return this._textValid;
+    }
+}
```

Comment:
End of file must have a blank line

Reply:
![image](https://user-images.githubusercontent.com/62728106/164026990-3cbf2373-0468-47a1-99aa-8bfd7d42bcfb.png)